skrf/calibration/deembedding.py

```diff
@@ -0,0 +1,452 @@
+'''
+.. module:: skrf.calibration.deembedding
+
+====================================================
+deembedding (:mod:`skrf.calibration.deembedding`)
+====================================================
+
+This module provides objects to implement de-embedding methods
+that are often used for on-wafer applications. 
+Each de-embedding method inherits from the common abstract base 
+class :class:`Deembedding`.

```

Comment:
It would be worth mentioning in the documentation somewhere why these objects differ from the calibration ones.